server/internal/iteminternal/components.go

```diff
@@ -81,5 +81,43 @@ func Components(it world.CustomItem) map[string]any {
 	if x, ok := it.(item.HandEquipped); ok {
 		builder.AddProperty("hand_equipped", x.HandEquipped())
 	}
+
+	x, y, z := calculateItemScale(it, float32(it.ImageSize()))
+
+	value := []float32{x, y, z}
```

Comment:
You may as well just make calculateItemScale return the size as a slice already, and also give a better name to value (e.g. itemScale)